src/fromager/packagesettings.py

```diff
@@ -219,6 +221,19 @@ class ProjectOverride(pydantic.BaseModel):
     """Remove requirement from pyproject.toml `[build-system] requires`
     """
 
+    requires_external: list[str] = Field(default_factory=list)
+    """Add / update Requires-External core metadata field
+
+    Each entry contains a string describing some dependency in the system
+    that the distribution is to be used. See
+    https://packaging.python.org/en/latest/specifications/core-metadata/#requires-external-multiple-use
+
+    .. note::
+       Fromager does not modify ``METADATA`` file, yet. Read the information
```

Comment:
What causes this new information to be written to the wheel?

Reply:
ah! ok, I was expecting this to be a new separate file, but including it there makes sense